src/test/java/com/exasol/exasoltestsetup/ExasolTestSetupTestBase.java

```diff
@@ -111,11 +111,12 @@ private void pingFromUdf(final ServiceAddress inDbAddress) throws SQLException {
     }
 
     @Test
-    void testMakeDatabaseServiceAvailableAtLocalhost() throws SQLException {
+    void testMakeDatabaseServiceAvailableAtLocalhost() throws SQLException, InterruptedException {
         final List<Integer> localPorts = this.testSetup.makeDatabaseTcpServiceAccessibleFromLocalhost(8001);
         assertThat(localPorts.size(), greaterThanOrEqualTo(1));
         final ConnectionTester connectionTester = new ConnectionTester(localPorts);
         createTcpServerInUdf();// blocking until received a connection or timeout
+        Thread.sleep(100);// wait a bit so that the connectionTester can receive
```

Comment:
can we add a `waitForConnectionTesterToBeReady` or something similar and remove the comment?